### PR TITLE
Textgroup_name_standardization

### PR DESCRIPTION
**Pull request by AlisonBabeu**
Updating textgroups so they don't include dates
Per issue #2563.

### CODE REVIEW

File: data/tlg0640/__cts__.xml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 <ti:textgroup xmlns:ti="http://chs.harvard.edu/xmlns/cts" urn="urn:cts:greekLit:tlg0640">
-  <ti:groupname xml:lang="lat">Alciphron Rhet., Soph., AD 2/3</ti:groupname>
+  <ti:groupname xml:lang="lat">Alciphron Rhet., Soph.</ti:groupname>
```
Comment (lcerrato) [inline data/tlg0640/__cts__.xml:2]: I wonder if we shouldn't spell out what "Rhet." and "Soph." are?

File: data/tlg2583/__cts__.xml [hunk 2]
```diff
@@ -1,3 +1,3 @@
 <ti:textgroup xmlns:ti="http://chs.harvard.edu/xmlns/cts" urn="urn:cts:greekLit:tlg2583">
-  <ti:groupname xml:lang="eng">Manetho Astrologus, 4th. century CE</ti:groupname>
+  <ti:groupname xml:lang="eng">Manetho Astrologus</ti:groupname>
```
Comment (lcerrato) [inline data/tlg2583/__cts__.xml:2]: Probably a Latin attribute?

**Review by lcerrato — CHANGES_REQUESTED** [review]

File: data/tlg0640/__cts__.xml (hunk 1 above)
Comment (AlisonBabeu) [reply data/tlg0640/__cts__.xml:2]: Probably. I also just looked this author up and the canonical authority name seems to be Alciphron, I think this is just from the TLG name, I should have changed it in the list.

File: data/tlg2583/__cts__.xml (hunk 2 above)
Comment (AlisonBabeu) [reply data/tlg2583/__cts__.xml:2]: I would think so, I will change that.

**Review by lcerrato — APPROVED** [review]

**Pull request merged**